servobuild.example

```diff
@@ -33,13 +35,16 @@ debug-mozjs = false
 
 # Android information
 [android]
-sdk = "/opt/android-sdk"
-ndk = "/opt/android-ndk"
-toolchain = "/opt/android-toolchain"
+# Defaults to "unspecified" and will instead pull from the 
```

Comment:
Not quoted-unspecified, just say "defaults to the values of `$ANDROID_NDK`, etc"